app/uk/gov/hmrc/serviceconfigs/persistence/DeploymentConfigRepository.scala

```diff
@@ -82,4 +95,7 @@ class DeploymentConfigRepository @Inject()(mongoComponent: MongoComponent)(impli
           equal("environment", environment.asString),
           in("name", names)
         )).toFuture().map(_ => ())
+
+  def deleteAll(): Future[Boolean] =
+    collection.deleteMany(new BasicDBObject()).toFuture.map(_.wasAcknowledged())
```

Comment:
I know this pattern is already around, but returning `wasAcknowledged` doesn't help much, and clients shouldn't have to check the boolean - you can just return `Future[Unit]` (`map(_ => ()`)

Reply:
Makes sense, will keep an eye out for this kind of thing going forward